micro-rdk/src/common/app_client.rs

```diff
@@ -165,10 +173,11 @@ impl AppClientBuilder {
     }
 }
 
+#[derive(Clone)]
```

Comment:
This is here from an older approach where we were instead going to `Clone` the app client. It could probably be backed out.